compiler/rustc_metadata/src/rmeta/decoder.rs

```diff
@@ -147,7 +165,11 @@ pub(super) struct DecodeContext<'a, 'tcx> {
 
 /// Abstract over the various ways one can create metadata decoders.
 pub(super) trait Metadata<'a, 'tcx>: Copy {
-    fn raw_bytes(self) -> &'a [u8];
+    fn blob(self) -> &'a MetadataBlob;
+    #[inline]
+    fn raw_bytes(self) -> &'a [u8] {
```

Comment:
I think it would be simpler to just remove this method, and let the callers rely on the `Deref` impl for `MetadataBlob`.

Reply:
Done!